pages/article_detail.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="page-wrapper">
+        <v-img
+            height="600"
+            lazy-src="https://via.placeholder.com/600"
+            :src="selectedArticle.urlToImage"
+        ></v-img>
+        <div class="pa-16">
+            <h1 class="page-heading">{{ selectedArticle.title || "--" }}</h1>
+            <div class="py-5 text-h6">
+                <div><span class="font-weight-light">Author</span>: {{ selectedArticle.author || "--" }}</div>
+                <div><span class="font-weight-light">Published On</span>: {{ formattedPublishDate || "--" }}</div>
+            </div>
+            <div>
+                <div class="text-subtitle-1">Article Detail:</div>
+                <p class="text-justify text-h6">{{ selectedArticle.content }}</p>
```

Comment:
is it a custom class that you have created?
vuetify as it's own text-center/left/right utility classes

Reply:
It is a util class from vuetify.